src/Me/Profile/Profile.css

```diff
@@ -0,0 +1,15 @@
+.profileLine {
```

Comment:
We're using [styled-components](https://styled-components.com/).

Can you convert the css to it? Here is a references for example: 
https://github.com/Coding-Coach/find-a-mentor/blob/339e0e3d38f981ebd915ad2afed490702615ed23/src/Me/Modals/Modal.js#L5-L14

Reply:
Sure! Converted